### DISCUSSION TITLE
Inverse Reinforcement Learning

### DISCUSSION THREAD

**Discussion by eirik-thorp** [rlworkgroup/garage#2228] [General]
Hi!

I am wondering if there is any plan to add any Inverse Reinforcement Learning algorithms in the near future?

From what I have experienced going through numerous code implementation, it seems that there is a dire need for some type of standard way of collecting and handling so-called expert trajectories. Something along the lines of the great Meta-RL implementations in torch. 

If there is anyone who has implemented some Inverse-RL algorithms on top of garage, please let me know!

**Comment by ryanjulian** [comment]
I think @krzentner is most likely to have the answer to this one
